build.sbt

```diff
@@ -24,55 +16,23 @@ Test / logBuffered := false
 run / fork := false
 Global / cancelable := false // ctrl-c
 
-val AkkaVersion = "2.6.12"
-val AkkaHttpVersion = "10.2.3"
-val AkkaManagementVersion = "1.0.9"
-val AkkaPersistenceJdbcVersion = "5.0.0"
-val AlpakkaKafkaVersion = "2.0.6"
-val AkkaProjectionVersion = "1.1.0"
 val ScalikeJdbcVersion = "3.5.0"
 
 enablePlugins(AkkaGrpcPlugin)
-
 enablePlugins(JavaAppPackaging, DockerPlugin)
 dockerBaseImage := "docker.io/library/adoptopenjdk:11-jre-hotspot"
 dockerUsername := sys.props.get("docker.username")
 dockerRepository := sys.props.get("docker.registry")
 ThisBuild / dynverSeparator := "-"
 
+libraryDependencies ++=
+  AkkaPlatform.httpServer ++
+  AkkaPlatform.clusterSharding ++
+  AkkaPlatform.clusterManagement ++
```

Comment:
we could argue that those three should be bundled together. Still, some people may not want part of it. 

Alternatively, we could have a `AkkaPlatform.microservices` that bundles all three.